### DISCUSSION TITLE
OpenApi examples for mock server

### DISCUSSION THREAD

**Discussion by Meyclem** [mockoon/mockoon#519] [Help] [answered]
Hi Mockoon community!

I'm trying to use Mockoon to generate a mock server for my API, based on an openapi spec file.

The API's endpoints are protected and need to have an `Authorization: Bearer <token>` header.
In my openapi spec, I have differents examples, like that:
```yaml
# This response is the 200 one on my GET /addresses endpoint.
    GetAddressesSuccess:
      description: Successful Get Addresses Request
      content:
        application/json:
          schema:
            # an array of addresses
            $ref: '#/components/schemas/AddressesResponse'
          examples:
            'Superman addresses':
              $ref: '#/components/examples/supermanAddresses'
            'No Address':
              $ref: '#/components/examples/noAddress'
```

So, what I would like to do is: get one different possible response for each example.

When I import the openapi file I have all my endpoints and responses code:
![image](https://user-images.githubusercontent.com/29123047/125273817-99d6c000-e30d-11eb-8175-87815eddc6bd.png)

But I can't fin a way to use my examples.

Does this functionality exist and I miss something or is it impossible?

**Comment by 255kb** [comment, marked as answer]
Hi! Mockoon is currently only supporting the `schema` property, not the `examples`. Do not hesitate to create a [new feature request](https://github.com/mockoon/mockoon/issues/new?assignees=&labels=&template=Feature_request.md) to track this.

**Reply by Meyclem** [reply]
Will do!

Thanks for the answer 🙏
